pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1546,7 +1546,7 @@ class Build {
                 def enableInstallers = Boolean.valueOf(buildConfig.ENABLE_INSTALLERS)
                 def enableSigner = Boolean.valueOf(buildConfig.ENABLE_SIGNER)
                 def useAdoptShellScripts = Boolean.valueOf(buildConfig.USE_ADOPT_SHELL_SCRIPTS)
-                def cleanWorkspace = Boolean.valueOf(buildConfig.CLEAN_WORKSPACE)
+                def cleanWorkspace = Boolean.valueOf(buildConfig.CLEAN_WORKSPACE_BEFORE_BUILD)
```

Comment:
```suggestion
                def cleanWorkspaceBefore = Boolean.valueOf(buildConfig.CLEAN_WORKSPACE_BEFORE_BUILD)
```

Reply:
I made a suggested change here that if accepted you'd want to propagate across the rest of the code (variable name change)

Reply:
I am fine with suggestion.